runtime/gc_modron_startup/mmparse.cpp

```diff
@@ -176,6 +176,35 @@ option_set(J9JavaVM* vm, const char* option, IDATA match)
 	return FIND_AND_CONSUME_VMARG2(match, option, NULL);
 }
 
+/**
+ * Find, consume and record an option from the argument list.
+ * Given an option string and the match type, find the argument in the to be consumed list.
+ * If not found, return success.
+ * If found, consume it, verify the memory value.
+ *
+ * @return OPTION_OK if option is found and consumed or option not present, OPTION_MALFORMED  if the option was malformed, OPTION_OVERFLOW if the option overflowed.
+ * @note value stored at address is invalid if failure returned
+ * @note optionIndex contains position of argument on command line if success returned, else -1
+ */
+static IDATA
+option_set_to_opt_args(J9JavaVM* vm, const char* option, IDATA* optionIndex, IDATA match, UDATA* address, J9VMInitArgs* args)
+{
+	IDATA element;
```

Comment:
a local should be initialized at the moment of declaration, so you can just merge this line with the one down below that initializes it